GeoHealthCheck/notifications.py

```diff
@@ -95,22 +95,20 @@ def do_email(config, resource, run, status_changed, result):
     if config['DEBUG']:
         server.set_debuglevel(True)
 
-    try:
-        if config['GHC_SMTP']['tls']:
+    if config['GHC_SMTP']['tls']:
+        LOGGER.debug('Authenticating via TLS')
+        try:
             server.starttls()
-    except Exception as err:
-        LOGGER.exception("Cannot connect to smtp: %s[:%s]: %s",
-                         config['GHC_SMTP']['server'],
-                         config['GHC_SMTP']['port'],
-                         err,
-                         exc_info=err)
+            server.login(config['GHC_SMTP']['username'],
+                         config['GHC_SMTP']['password'])
+        except Exception as err:
+            LOGGER.exception("Cannot connect/authenticate to smtp: %s:%s: %s",
+                             config['GHC_SMTP']['server'],
+                             config['GHC_SMTP']['port'],
+                             err,
+                             exc_info=err)
         return
```

Comment:
Shouldn't `return` be indented? Now we would always return in case of TLS, exception or not.

Reply:
fixed.